online-shop/src/app/components/product-view/product-view.component.html

```diff
@@ -1,11 +1,17 @@
-<button *ngIf="product && hasAuthorisationOfAdmin"
-        [routerLink]="['/', 'edit-product', product.id]">EDIT
+<button (click)="goBack()">
+  <mat-icon role="img"
+            aria-hidden="true"
+            data-mat-icon-type="font">home
+  </mat-icon>
+</button>
+<button *ngIf="selectedProduct && hasAuthorisationOfAdmin"
+        [routerLink]="['/', 'edit-product', selectedProduct.id]">
+  <mat-icon>edit</mat-icon>
 </button>
-<button (click)="goBack()">CANCEL</button>
 <router-outlet></router-outlet>
-<button (click)="deleteProduct()" *ngIf="hasAuthorisationOfAdmin">DELETE</button>
-<p *ngIf="product">{{this.product.name}}</p>
-<p *ngIf="product">Avem urmatorul id {{this.product.id}}</p>
-<p *ngIf="product">{{this.product.description}}</p>
-<p *ngIf="product">{{this.product.price}}</p>
-<img *ngIf="product" src="{{this.product.image}}" alt="imagine pe care nu o gasim">
+<button (click)="deleteProduct()" *ngIf="hasAuthorisationOfAdmin"><mat-icon>delete_forever</mat-icon></button>
+<p *ngIf="selectedProduct">{{this.selectedProduct.name}}</p>
+<p *ngIf="selectedProduct">Avem urmatorul id {{this.selectedProduct.id}}</p>
+<p *ngIf="selectedProduct">{{this.selectedProduct.description}}</p>
+<p *ngIf="selectedProduct">{{this.selectedProduct.price}}</p>
+<img *ngIf="selectedProduct" src="{{this.selectedProduct.image}}" alt="imagine pe care nu o gasim">
```

Comment:
By now it's fine, but in future please use only english. The reason is that there are big chances to work in future with foreign colleagues, and they won't understand :)